tapir/shifts/views.py

```diff
@@ -290,6 +295,24 @@ def form_valid(self, form):
 
             return response
 
+    def send_shift_missed_email(self):
```

Comment:
Just a comment: This is part of a bigger thing really, but we don't really have a good story for where to put this kind of business logic. We might eventually have more places where we send this email, and then we'll probably move it into the model, but having super-fat models isn't great either. Some projects introduce a controller layer at some point, but I think that would be too much...